pkg/apis/serving/v1beta1/transformer_custom.go

```diff
@@ -66,6 +66,15 @@ func (c *CustomTransformer) GetStorageUri() *string {
 // GetContainers transforms the resource into a container spec
 func (c *CustomTransformer) GetContainer(metadata metav1.ObjectMeta, extensions *ComponentExtensionSpec, config *InferenceServicesConfig) *v1.Container {
 	container := &c.Containers[0]
+	var argumentPredictorHost string
+	annotations := utils.Filter(metadata.Annotations, func(key string) bool {
+		return !utils.Includes(constants.ServiceAnnotationDisallowedList, key)
+	})
+	if value, ok := annotations[constants.RawDeploymentAnnotationKey]; ok && value == "true" {
```

Comment:
I think here should be the same for raw deployment, namespace isn't strictly needed if all the resources are in the same namespace but better keeps the logic consistent.